From c93b30daa373f8f11b28c5d020976361c2af589e Mon Sep 17 00:00:00 2001
From: Faith Ekstrand <faith.ekstrand@collabora.com>
Date: Thu, 19 Oct 2023 03:07:16 -0500
Subject: [PATCH 579/834] nvk: Advertise VK_EXT_attachment_feedback_loop_layout

We don't need to do anything with the layout.

Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25808>
---
 src/nouveau/vulkan/nvk_physical_device.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/nouveau/vulkan/nvk_physical_device.c b/src/nouveau/vulkan/nvk_physical_device.c
index ff72046fab2..fd271e1c27d 100644
--- a/src/nouveau/vulkan/nvk_physical_device.c
+++ b/src/nouveau/vulkan/nvk_physical_device.c
@@ -98,6 +98,7 @@ nvk_get_device_extensions(const struct nv_device_info *info,
       .KHR_variable_pointers = true,
       .KHR_workgroup_memory_explicit_layout = true,
       .EXT_4444_formats = true,
+      .EXT_attachment_feedback_loop_layout = true,
       .EXT_border_color_swizzle = true,
       .EXT_buffer_device_address = true,
       .EXT_conditional_rendering = true,
@@ -265,6 +266,9 @@ nvk_get_device_features(const struct nv_device_info *info,
       .formatA4R4G4B4 = true,
       .formatA4B4G4R4 = true,
 
+      /* VK_EXT_attachment_feedback_loop_layout */
+      .attachmentFeedbackLoopLayout = true,
+
       /* VK_EXT_border_color_swizzle */
       .borderColorSwizzle = true,
       .borderColorSwizzleFromImage = false,
-- 
2.42.0

